docs/setup/setup.md

```diff
@@ -62,18 +62,19 @@ The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/cert
 
 #### Run as a binary
 
++ First, you can run `gcc -version` check the gcc for install
```

Comment:
Firstly, make sure gcc is already installed on your host. You can verify it via:

```
gcc -version
```